.php_cs

```diff
@@ -0,0 +1,26 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
```

Comment:
Okay, now filtering *.php and *.php. Also added a new section in CONTRIBUTING.md about coding standards.